docs/DeveloperGuide.adoc

```diff
@@ -345,6 +345,61 @@ image::UndoRedoActivityDiagram.png[width="200"]
 **Cons:** Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things. +
 // end::undoredo[]
 
+// tag::findClose[]
+=== Fuzzy finding mechanism
+The fuzzy finding mechanism is powered by https://en.wikipedia.org/wiki/Levenshtein_distance[Levenshtein distance],
+a string heuristic for measuring the difference between two character sequences. It can be informally viewed as the number
```

Comment:
Perhaps "string heuristic" can be replaced with non-technical jargon, e.g. simply "technique"?